apps/namadillo/src/App/Layout/SyncIndicator.tsx

```diff
@@ -7,17 +7,43 @@ import {
 import { useAtomValue } from "jotai";
 import { twMerge } from "tailwind-merge";
 
+const formatError = (
```

Comment:
think we should bring this down into the component itself?

Reply:
oh, if we do it, it'll recreate the function every time, but we don't need it :)

Reply:
TIL lol...I always just throw it in the component for cleanliness but to save bytes makes total sense to me! 👍